src/config.rs

```diff
@@ -1019,6 +1031,38 @@ impl Config {
     }
 }
 
+/// Window Dimensions
+///
+/// Initial window dimensions in characters
+#[derive(Debug, Deserialize)]
+pub struct Dimensions {
```

Comment:
It would be nice for the `Options` to use this same `Dimensions` type instead of tracking `lines` and `columns` separately.